f8a_report/helpers/db_gateway.py

```diff
@@ -105,3 +106,40 @@ def retrieve_ingestion_results(self, start_date, end_date) -> dict:
         data = json.dumps(self.cursor.fetchall())
 
         return data
+
+
+class TokenValidationQueries(Postgres):
+    """Snyk Token Validation Queries."""
+
+    def __init__(self):
+        """Class Constructor."""
+        super().__init__()
+
+    def get_registered_user_tokens(self) -> dict:
+        """Tokens for registered users."""
+        try:
+            get_registered_user_sql = \
+                sql.SQL("select user_id, snyk_api_token from user_details where status=\'%s\'")
+            self.cursor.execute(get_registered_user_sql.as_string(self.conn)
+                                % UserStatus.REGISTERED.name)
+            result = self.cursor.fetchall()
+            user_id_token_dict = {row[0]: row[1] for row in result}
+            return user_id_token_dict
+        finally:
+            self.conn.close()
+
+    def update_users_to_unregistered(self, unregistered_users: list):
```

Comment:
Should we update `updated_date` field for registered users as well?

Reply:
I dont think so, as we are not updating anything for registered users and also the job will run for all registered users. It is not an interval based check